product-backend/src/main/java/org/kainos/ea/api/AuthService.java

```diff
@@ -0,0 +1,35 @@
+package org.kainos.ea.api;
+
+import org.kainos.ea.cli.RequestEmployee;
+import org.kainos.ea.client.FailedToCreateNewEmployeeException;
+import org.kainos.ea.client.FaliedToCreateEmployeeWrongInputException;
+import org.kainos.ea.db.AuthDao;
+import org.kainos.ea.validator.RegisterValidator;
+
+import java.sql.SQLException;
+
+public class AuthService {
+
+    public AuthService(AuthDao authDao) {
+        this.authDao = authDao;
+    }
+
+
+    private AuthDao authDao;
+
+    public int createNewEmployee(RequestEmployee input) throws FailedToCreateNewEmployeeException, FaliedToCreateEmployeeWrongInputException {
```

Comment:
Could be void
```suggestion
    public void createNewEmployee(RequestEmployee input) throws FailedToCreateNewEmployeeException, FaliedToCreateEmployeeWrongInputException {
```

Reply:
Yes, but based on that id that it returns already quite a lot of dependencies created. 
 I would suggest keeping it that way unless there is a good reason to change it. 
 What would you say?